client/src/AppBundle/Resources/translations/report-prof-deputy-costs.en.yml

```diff
@@ -134,7 +134,7 @@ summaryPage:
     totalCosts: Total costs
 
 deletePage:
-  htmlTitle: "Deputy report - Deputy costs | GOV.UK"
+  htmlTitle: "Deputy costs - Complete the deputy report | GOV.UK"
```

Comment:
Remove deputy cost?

Reply:
In the context of the page the deputy costs title makes sense to me.